containers/monitoring-prometheus-image/Dockerfile

```diff
@@ -0,0 +1,41 @@
+# SPDX-License-Identifier: MIT
+
+#     Copyright (c) 2024 SUSE LLC
+
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon.
+
+#!BuildTag: uyuni/prometheus:latest
+
+FROM suse/sle15:15.6
```

Comment:
Should the base image be this one for both Uyuni and SUSE Manager?

Reply:
I think Prometheus package is not available in Leap. Not sure what other base image we should use for Uyuni.

Reply:
@witekest https://smelt.suse.de/maintained/?q=golang-github-prometheus-prometheus&with_debug=1 from  `SUSE:SLE-15-SP1:Update` we also  release to  Leap

Reply:
I have changed the base image to `bci/bci-base:15.6`.